src/lib/client/localStorage/users.ts

```diff
@@ -9,10 +9,16 @@ import {
 export const USERS_STORAGE_KEY = "users";
 
 export type User = {
-  displayName: string;
-  encryptionPublicKey: string;
-  twitterUsername?: string;
-  telegramUsername?: string;
+  name: string; // User's display name
+  encPk: string; // User's encryption public key
+  x?: string; // User's Twitter username
```

Comment:
Slightly confusing variable name for twitter

Reply:
Its the shortest one that makes sense